changelog/#6694 .bug.rst

```diff
@@ -0,0 +1,2 @@
+the current version of ``sqlalchemy`` has been pinned < ``2.0`` till investigation is being done on new
```

Comment:
The filename should have no spaces in it, nor a #.

For now, I think a trivial category would be enough.